CHANGELOG.md

```diff
@@ -1,14 +1,13 @@
 # Changelog
 
-Used to document all changes from previous releases and collect changes 
-until the next release.
+Used to document all changes from previous releases and collect changes until the next release.
 
-# Latest changes in master
+# Version 0.0.7
```

Comment:
Are you going to make a release once these changes are merged? If not, I guess I would rather not yet assign a version number

Reply:
I already have a release prepared and tested via PyPI test.